### DISCUSSION TITLE
Sync folder/album to uncategorised

### DISCUSSION THREAD

**Discussion by Moumirrai** [ente-io/ente#1360] [Enhancements]
As of now, when I sync abum from mobile to ente so the photos are automatically assigned to their album. I would really like to see which of my photos are or are not included elsewhere than in my main Camera folder. I think this could be solved by making some sort of "uncategorized" option when setting up the folder sync.

**Comment by vishnukvmd** [comment]
Hey, by default the photos will land in the folders they belong to on your device.

We do have an Uncategorized section (within the second tab), that contains photos that aren't attached to any album.

Is this what you're looking for?

**Reply by tanguyduval** [reply]
The idea would be to simply add an option in " Uncategorized section" to select albums that do not count as "categorized". 

By default the sync album from the phone Camera gallery shall be considered as uncategorized.

This would be a very great and distinctive feature

**Comment by iwcvanderlinden** [comment]
Indeed this would be useful. The uncategorized section is a useful features, but clashes with automatically tagging pictures based on the upload origin (i.e. camera roll). Tagging with upload origin is useful as well. For instance, one might remember what device was used to take a picture and use this when searching. But without the suggested feature, this make the uncategorized section redundant: no photos are ever uncategorized. If we could exclude certain tags (camera roll, digital camera, etc) from this filter, we'd be able to find all photos we've uploaded but not manually categorized into albums under uncategorized.

**Comment by ghost** [comment]
The camera roll itself isn't really an intentional album I created so it would be nice to see it go to uncategorized instead of to an album like "recents".
